server_addon/deadline/server/settings/publish_plugins.py

```diff
@@ -306,36 +349,41 @@ class PublishPluginsModel(BaseSettingsModel):
         default_factory=ValidateExpectedFilesModel,
         title="Validate Expected Files"
     )
-    MayaSubmitDeadline: MayaSubmitDeadlineModel = SettingsField(
-        default_factory=MayaSubmitDeadlineModel,
-        title="Maya Submit to deadline")
-    MaxSubmitDeadline: MaxSubmitDeadlineModel = SettingsField(
-        default_factory=MaxSubmitDeadlineModel,
-        title="Max Submit to deadline")
-    FusionSubmitDeadline: FusionSubmitDeadlineModel = SettingsField(
-        default_factory=FusionSubmitDeadlineModel,
-        title="Fusion submit to Deadline")
-    NukeSubmitDeadline: NukeSubmitDeadlineModel = SettingsField(
-        default_factory=NukeSubmitDeadlineModel,
-        title="Nuke Submit to deadline")
-    HarmonySubmitDeadline: HarmonySubmitDeadlineModel = SettingsField(
-        default_factory=HarmonySubmitDeadlineModel,
-        title="Harmony Submit to deadline")
     AfterEffectsSubmitDeadline: AfterEffectsSubmitDeadlineModel = (
         SettingsField(
             default_factory=AfterEffectsSubmitDeadlineModel,
-            title="After Effects to deadline"
+            title="After Effects to deadline",
+            section="Hosts"
         )
     )
-    CelactionSubmitDeadline: CelactionSubmitDeadlineModel = SettingsField(
-        default_factory=CelactionSubmitDeadlineModel,
-        title="Celaction Submit Deadline")
     BlenderSubmitDeadline: BlenderSubmitDeadlineModel = SettingsField(
         default_factory=BlenderSubmitDeadlineModel,
         title="Blender Submit Deadline")
+    CelactionSubmitDeadline: CelactionSubmitDeadlineModel = SettingsField(
```

Comment:
I think the order was determined. I don't think the change of the order should not happen in this PR.

Reply:
https://github.com/ynput/ayon-core/pull/17